{{cookiecutter.plugin_name}}/README.md

```diff
@@ -4,87 +4,70 @@
 
 This `nomad` plugin was generated with `Cookiecutter` along with `@nomad`'s [`cookiecutter-nomad-plugin`](https://github.com/FAIRmat-NFDI/cookiecutter-nomad-plugin) template.
 
-
 ## Development
 
-If you want to develop locally this plugin, clone the project and in the plugin folder, create a virtual environment (you can use Python 3.9, 3.10, or 3.11):
+If you want to locally develop this plugin, clone the project.
+
 ```sh
 git clone https://github.com/{{cookiecutter.github_username}}/{{cookiecutter.plugin_name}}.git
 cd {{cookiecutter.plugin_name}}
-python3.11 -m venv .pyenv
-. .pyenv/bin/activate
 ```
 
-Make sure to have `pip` upgraded:
-```sh
-pip install --upgrade pip
-```
+We highly recommend using `uv` to manage the project. Consider installing `uv` by following the official installation [guide](https://docs.astral.sh/uv/getting-started/installation/#installing-uv).
```

Comment:
Please, do not send people people to read some other documentation in another package management. Can you add a better explanation + step-by-step as it was before? Think that people is not going to open that uv guide (not even me), so is `uv` replacing the virtual environment? Can you summarize how uv manages projects in 1 or 2 lines?